tests/foundry/golden/AssertTest.testFail_assert_true.expected

```diff
@@ -882,7 +882,13 @@ module SUMMARY-ASSERTTEST.TESTFAIL-ASSERT-TRUE:197B53D8A0C19A3B62F2C833DF85695B4
 endmodule
 1 Failure nodes. (0 pending and 1 failing)
 
+<<<<<<< HEAD
+1 Failure nodes. (0 frontier and 1 stuck)
+
+Stuck nodes:
+=======
 Failing nodes:
+>>>>>>> origin/master
 
```

Comment:
What's going on here?

Reply:
I guess these files are no longer used, so running the update didn't overwrite the failed automerge version. I removed all these expected files in favor of the ones at the location used by the pytest tests.